src/views/accounts/RiskDetect/AccountCheckTaskList.vue

```diff
@@ -44,15 +44,15 @@ export default {
             formatterArgs: {
               async: false,
               drawer: false,
-              prevantClick: true
+              preventClick: true
             }
           },
           nodes: {
             formatter: AmountFormatter,
             formatterArgs: {
               async: false,
               drawer: false,
-              prevantClick: true
+              preventClick: true
             }
           },
           secret_strategy: {
```

Comment:
The formatting arguments of the AmountFormatter seem to be consistent in both files. However, some changes have been made:

1. The `prevantClick` flag from line 66 has changed back to just "click" on line 72 while all other parameters remained same.

This should not create many issues except for if you're using this component in another context that requires certain flags based on user behavior (like preventing clicks) which was not clear originally. 

For optimization purposes, I suggest ensuring consistency across components to maintain a clean structure and avoid breaking future updates without notice.